src/helpers/dom/classList.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @class ClassList
+ */
+class ClassList {
+  constructor(element) {
+    this._element = element;
+  }
+
+  contains(value) {
+    const className = this._element.className;
+    const classLen = className.length;
+    const valueLen = value.length;
+    const isLenDiffer = classLen >= valueLen;
+    let index = classLen > 0 && valueLen > 0 && isLenDiffer ? className.indexOf(value) : -1;
```

Comment:
No, it can return boolean or number, please, check what happens with boolean values.